doc/rst/usingchapel/chplenv.rst

```diff
@@ -396,14 +396,18 @@ CHPL_COMM
         ======= ============================================
         none    only supports single-locale execution
         gasnet  use the GASNet-based communication layer
+        ofi     use the (preliminary) libfabric-based communication layer
         ugni    Cray-specific native communication layer
         ======= ============================================
 
    If unset, ``CHPL_COMM`` defaults to ``none`` in most cases.  On Cray XE
    and XC systems it defaults to ``ugni``.  On Cray CS systems it defaults
    to ``gasnet``.  See :ref:`readme-multilocale` for more information on
-   executing Chapel programs using multiple locales.  See :ref:`readme-cray`
-   for more information about Cray-specific runtime layers.
+   executing Chapel programs using multiple locales.
+   See :ref:`readme-libfabric` for more information about the ofi
+   communication layer.
+   See :ref:`readme-cray` for more information about Cray-specific runtime
+   layers.
```

Comment:
Remove linebreaks here?

Reply:
Sure.  I originally started doing that because it doesn't affect the formatted output, but it might allow for fewer changed lines when someone comes along in the future and changes a sentence.  But obviously I didn't stick to it.